baseapp_drf_view_action_permissions/models.py

```
@@ -37,16 +39,14 @@ def get_permission_list(self, user_exclude_perms=[]):
-                perms = {*perms, *perm_set}
+                perms |= {"%s.%s" % (ct, name) for ct, name in group_perms}
 
             group_perms = self.permissions.filter(~Q(codename__in=excluded_perms)).values_list(
                 "content_type__app_label", "codename"
             )
-            perm_set = {"%s.%s" % (ct, name) for ct, name in group_perms}
-            perms = {*perms, *perm_set}
+            perms |= {"%s.%s" % (ct, name) for ct, name in group_perms}
         except Exception:
-            return perms
+            pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)